DemoApp/app/src/main/java/com/colortv/demo/VideoActivity.java

```diff
@@ -0,0 +1,106 @@
+package com.colortv.demo;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.graphics.Color;
+import android.media.MediaPlayer;
+import android.net.Uri;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.util.Log;
+import android.view.MotionEvent;
+import android.view.View;
+import android.widget.MediaController;
+import android.widget.VideoView;
+
+import com.colortv.android.ColorTvContentRecommendationListener;
+import com.colortv.android.ColorTvError;
+import com.colortv.android.ColorTvSdk;
+import com.colortv.android.TrackingEventType;
+
+public class VideoActivity extends Activity {
+
+    private VideoView videoView;
+    private String videoId;
+    private String currentPlacement = "TestContent";
+
+    private ColorTvContentRecommendationListener loadContentRecommendationListener = new ColorTvContentRecommendationListener() {
+
+        @Override
+        public void onLoaded(String placement) {
+            currentPlacement = placement;
+        }
+
+        @Override
+        public void onError(String placement, ColorTvError error) {
+            Log.e(VideoActivity.class.getSimpleName(), "Error while fetching ContentRecommendation for placement " +
```

Comment:
Please create a constant `private static final String TAG = VideoActivity.class.getSimpleName()`, because it looks messy otherwise.